app/javascript/packs/speaker_form.js

```diff
@@ -23,8 +22,22 @@ document.addEventListener('DOMContentLoaded', () => {
         addDeleteButtonListener(div.querySelector('.remove_talk_field'));
         return false;
     });
+}
+
+const removeTalkFieldListener = () => {
```

Comment:
これだとTalkFieldListenerを削除するように読めてしまうので、メソッド名変えたいところ
英語としては変だけど、他の命名でも `addDeleteButtonListener` みたいなのがあったりするので、

```suggestion
const addRemoveTalkFieldListener = () => {
```
とかかなぁ

Reply:
あー、確かに。

Reply:
https://github.com/cloudnativedaysjp/dreamkast/pull/2287/commits/01c16ad3cd94dc4b64d4fd7146a9836f8b7ad0e2 でリネームしてみた。